src/main/webapp/app/shared/tab-bar/tab-bar.scss

```diff
@@ -0,0 +1,55 @@
+.tab-bar {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0.5rem 0;
+    margin-bottom: 0.5rem;
+
+    border-bottom: 1px solid var(--overview-border-color);
+
+    a {
+        display: inline-block;
+        position: relative;
+        color: var(--tab-bar-color);
+        text-decoration: none !important; // override bootstrap
```

Comment:
Please just use a more specific CSS selector than Bootstrap uses to "override" their `text-decoration` if possible

Reply:
Fixed, I had to change something in the global.scss, where !important was used. I've made it so that links with the class "tab-link" are no longer affected by that.